components/calendly_v2/actions/create-scheduling-link/create-scheduling-link.mjs

```diff
@@ -0,0 +1,35 @@
+import calendly from "../../calendly_v2.app.mjs";
+
+export default {
+  key: "calendly-create-scheduling-link",
+  name: "Create a Scheduling Link",
+  description: "Creates a single-use scheduling link. [See the docs](https://calendly.stoplight.io/docs/api-docs/b3A6MzQyNTM0OQ-create-single-use-scheduling-link)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    calendly,
+    owner: {
+      propDefinition: [
+        calendly,
+        "eventType",
+      ],
+      label: "Owner",
+    },
+    maxEventCount: {
+      propDefinition: [
+        calendly,
+        "maxEventCount",
+      ],
+    },
+  },
+  async run({ $ }) {
+    let params = {};
+    params.max_event_count = this.maxEventCount;
+    params.owner = this.owner;
+    params.owner_type = "EventType";
+
+    let response = await this.calendly.createSchedulingLink(params);
```

Comment:
`response` can be a constant as well. 

For actions that use `@pipedream/platform` axios, we want to pass in `$` so that we get more useful debugging info if an error is thrown. 
https://pipedream.com/docs/pipedream-axios/#using-pipedream-platform-axios-in-component-actions
```suggestion
    const response = await this.calendly.createSchedulingLink(params, $);
```

Reply:
Issue addressed